### PR TITLE
fix: Governance Proposal Screen ui break in mobile and web

### PR DESCRIPTION
**Pull request by ChiragPansuriya-iView**
fix the issue of the Governance Proposal ScreenUI break in mobile and web.

simulator_screenshot_83E4810D-251B-4D65-892D-5FC2E52FFB05
Screenshot 2024-04-08 at 12 39 27 PM

### CODE REVIEW

File: packages/screens/Governance/GovernanceProposal/GovernanceVoteHeader/components/VoteTimeDetailsBox.tsx [hunk 1]
```diff
@@ -46,19 +48,41 @@ export const VoteTimeDetailsBox: React.FC<{
...
-        <VoteTimeText
-          label="Voting End"
-          subLabel={`${voting_end_time.slice(0, 10)} ${voting_end_time.slice(11, 16)} UTC`}
-        />
+        {!isMobile && (
+          <>
+            <Separator horizontal />
+            <VoteTimeText
+              label="Voting Start"
+              subLabel={`${voting_start_time.slice(0, 10)} ${voting_start_time.slice(11, 16)} UTC`}
```
Comment (omniwired) [inline packages/screens/Governance/GovernanceProposal/GovernanceVoteHeader/components/VoteTimeDetailsBox.tsx:56]: is there any other less hacky way to do this?
Maybe get the date as an object and format it in another way?

Comment (n0izn0iz) [reply packages/screens/Governance/GovernanceProposal/GovernanceVoteHeader/components/VoteTimeDetailsBox.tsx:56]: yeah, probably should use moment

**Review by n0izn0iz — CHANGES_REQUESTED** [review]

**Review by omniwired — APPROVED** [review]
:shipit:

**Review by n0izn0iz — APPROVED** [review]
LGTM

**Pull request merged**